docs/slides/innersource-summit.md

```diff
@@ -0,0 +1,237 @@
+# The InnerSource Program Office (ISPO) Working Group
+<!---  Slide 2: Opening Statement --->
+
+![bg right](../assets/innersource-commons.svg)
+
+The InnerSource Program Office (ISPO) Working Group develops guidance and products for enterprise InnerSource teams, not individuals or projects.
```

Comment:
1. Shortened.
2. Dropped "not individuals or projects" since it carries a negative connotation and is less inclusive.

Even someone not part of an enterprise InnerSource team could benefit from following our working group's progress. With that said, our focus is stated explicitly in this sentence.

```suggestion
The InnerSource Program Office (ISPO) Working Group creates resources for enterprise InnerSource teams.
```